lib/segment/src/index/field_index/geo_hash.rs

```diff
@@ -221,66 +223,80 @@ fn check_polygon_intersection(geohash: &str, polygon: &GeoPolygon) -> bool {
     rect.intersects(&polygon)
 }
 
+fn create_hashes(
+    mapping_fn: impl Fn(usize) -> Option<Vec<GeoHash>>,
+) -> Result<Vec<GeoHash>, OperationError> {
```

Comment:
We have the alias `pub type OperationResult<T> = result::Result<T, OperationError>;` to shorten things up :)

Reply:
Thanks @agourlay! I made an amendment!